crates/gwt-tauri/src/python/chroma_index_runner.py

```
@@ -0,0 +1,495 @@
+            if isinstance(data, dict) and "description" in data:
+                return str(data["description"])
+        except (json.JSONDecodeError, ValueError):
+            pass
+        return file_path.name
+
+    # Python: module docstring or first comment
+    if suffix == ".py":
+        for line in lines[:10]:
+            stripped = line.strip()
+            if stripped.startswith('"""') or stripped.startswith("'''"):
+                text = stripped[3:].rstrip('"""').rstrip("'''").strip()
```

Ruff (0.15.2)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)